### PR TITLE
HW_Models: NRF_PPI: Enable ppi task and event for TIMER0 CH3

### PR DESCRIPTION
**Pull request by jrotkiewicz**
Used in Mynewt integration with Babblesim.

Signed-off-by: Jakub Rotkiewicz <jakub.rotkiewicz@codecoup.pl>

### CODE REVIEW

File: src/HW_models/NRF_RADIO.c [hunk 1]
```diff
@@ -654,6 +654,7 @@ void nrf_radio_timer_triggered(){
   } else if ( radio_state == RXDISABLE ){
     radio_state = DISABLED;
     Timer_RADIO = TIME_NEVER;
+    NRF_RADIO_regs.STATE = DISABLED;
```
Comment (thoh-ot) [inline src/HW_models/NRF_RADIO.c:663]: I would move this up to be under `radio_state` such that it matches the order used other places in this function.

@aescolar If the assignment to `NRF_RADIO_regs.STATE` is missing here, isn't it then also missing in L650?

**Review by thoh-ot — CHANGES_REQUESTED** [review]

File: src/HW_models/NRF_RADIO.c (hunk 1 above)
Comment (aescolar) [reply src/HW_models/NRF_RADIO.c:663]: @thoh-ot it seems to be have been a silly omission in both places yes.
@cvinayak I guess you agree that this should have been set?
@jrotkiewicz would you be so kind as to correct it in both places and move it up one line as requested by @thoh-ot ?
With that I think we can merge it

**Comment by aescolar** [comment]
Thanks @jrotkiewicz. I will look into it in the following days

**Comment by jrotkiewicz** [comment]
My friend @mkasenberg found out some additional changes required for BabbleSim to work on Mynewt. I've added them above. Please review, thank you.

**Comment by aescolar** [comment]
Thanks. Merged.

**Pull request closed without merging**